src/SourceBuild/patches/aspnetcore/0001-Rename-SourceBuildUseMonoRuntime-property-which-is-n.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jo Shields <joshield@microsoft.com>
+Date: Mon, 23 Sep 2024 10:11:07 -0400
+Subject: [PATCH] Rename SourceBuildUseMonoRuntime property, which is not SB
+ specific
+
+This is a coordinated cross-repo change, and might break the build until
+all four PRs are merged. The other three are:
+
+Ref: https://github.com/dotnet/source-build/issues/4165
```

Comment:
Nit:  Convention per https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines is to use Backport:

```suggestion
Backport: https://github.com/dotnet/source-build/issues/4165
```